### PR TITLE
docs(protocol): typo fix

### PR DESCRIPTION
**Pull request by buldazer23**
Typo fix

### CODE REVIEW

**Review by 2manslkh — DISMISSED** [review]
LGTM

File: packages/bridge-ui-v2/config/schemas/configuredBridges.schema.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
             "required": [
               "bridgeAddress",
               "erc20VaultAddress",
+              "etherVaultAddress",
```
Comment (d1onys1us) [inline packages/bridge-ui-v2/config/schemas/configuredBridges.schema.json:44]: why do we need this here? cc @KorbinianK

Comment (buldazer23) [reply packages/bridge-ui-v2/config/schemas/configuredBridges.schema.json:44]: "addresses": {
            "type": "object",
            "properties": {
              "bridgeAddress": {
                "type": "string"
              },
              "erc20VaultAddress": {
                "type": "string"
              },
              "etherVaultAddress": {
                "type": "string"
              },
              "erc721VaultAddress": {
                "type": "string"
              },
              "erc1155VaultAddress": {
                "type": "string"
              },
              "crossChainSyncAddress": {
                "type": "string"
              },
              "signalServiceAddress": {
                "type": "string"
              }
            },
            "required": [
              "bridgeAddress",
              "erc20VaultAddress",
              **"etherVaultAddress",**
              "erc721VaultAddress",
              "erc1155VaultAddress",
              "crossChainSyncAddress",
              "signalServiceAddress"
            ],
              I thought it should be there due to the code sequence at the top, but if this addition doesn't make sense, I can correct it.

Comment (dantaik) [inline packages/bridge-ui-v2/config/schemas/configuredBridges.schema.json:44]:
```suggestion
```

**Review by dantaik — APPROVED** [review]

**Review by davidtaikocha — APPROVED** [review]

**Review by dionysuzx — APPROVED** [review]

**Pull request merged**